### PR TITLE
Added condition to check if the job is activated to prevent the error "@esx_service/server/main.lua:52: attempt to index a nil value (field '?')"

### PR DESCRIPTION
**Pull request by alvinveroy**
Patched the function with this code that adds checking if the Job is activated.

```
ESX.RegisterServerCallback('esx_service:isInService', function(source, cb, name)
	local isInService = false

	if InService[name] ~= nil then -- checks if the job is activated for service
		if InService[name][source] then
			isInService = true
		end
	end

	cb(isInService)
end)
```

### CODE REVIEW

**Comment by HezimX** [comment]
not good at coding but just wondering should there be a then at the end of     

if In Service[name] ~= nil -- checks if the job is activated for service     

koz i get an error without

**Comment by alvinveroy** [comment]
> not good at coding but just wondering should there be a then at the end of
> 
> if In Service[name] ~= nil -- checks if the job is activated for service
> 
> koz i get an error without

I accidentally removed the then when I added the comment, send the new request again. Thanks for pointing it out.

File: server/main.lua [hunk 1]
```diff
@@ -49,8 +49,10 @@ end)
 ESX.RegisterServerCallback('esx_service:isInService', function(source, cb, name)
 	local isInService = false
 
-	if InService[name][source] then
-		isInService = true
+	if InService[name] ~= nil then -- checks if the job is activated for service
```
Comment (Dislaik) [inline server/main.lua:52]:
```suggestion
	if InService[name] ~= nil then
```
and will be go

**Comment by gabrielcalderon** [comment]
Looks good! 👍

**Pull request merged**